### DISCUSSION TITLE
Where to add special config directives

### DISCUSSION THREAD

**Discussion by Budman1758** [NginxProxyManager/nginx-proxy-manager#2548] [Q&A]
Hello. I am trying to get a special config set up to allow curl commands to work correctly. I have an [ntfy](https://docs.ntfy.sh/config/#behind-a-proxy-tls-etc) self hosted server and using nginx PM to proxy it. As stated in the ntfy docs you need a couple special statements in the proxy config to allow curl without https.

Can anybody give some pointers as to how to get this working? 

Explanation of what I'm refering to at this link.   https://docs.ntfy.sh/config/#nginxapache2caddy

Thanks for any pointers here.

**Comment by Budman1758** [comment]
Anybody?? Surely this is doable.

**Comment by woiwoiwoiwoi** [comment]
Hey, did you ever manage to solve this?

**Reply by Budman1758** [reply]
Nope.  Absolutely no help here at all.

**Reply by woiwoiwoiwoi** [reply]
That's a bummer. Have you considered [this fork](https://github.com/ZoeyVid/NPMplus), which seems very active? Myself, I'm unsure about it.

**Reply by Budman1758** [reply]
Actually I haven't even messed with it for quite some time.  I might try that fork and see what's up though.  Might be a while before I do.  Probably after the holidays.
